### PR TITLE
Refactor unit tests to move everything under tests.

### PR DESCRIPTION
**Pull request by diptorupd**

### CODE REVIEW

**Comment by xaleryb** [comment]
CI is finished successfully for this PR

https://teamcity-or.intel.com/viewLog.html?buildId=6587342&buildTypeId=Sat_NewCi_IndividualPackages_Pydppl_Development_Python37_Windows_Deployment

File: dpctl/tests/test_sycl_queue_manager.py [hunk 1]
```diff
@@ -19,51 +19,51 @@
...
+    def test_dpctl_dump_device_info (self):
         q = dpctl.get_current_queue()
         try:
             q.get_sycl_device().dump_device_info()
         except Exception:
             self.fail("Encountered an exception inside dump_device_info().")
 
 @unittest.skipIf(not dpctl.has_sycl_platforms(), "No SYCL platforms available")
-class TestDPPLIsInDPPLCtxt (unittest.TestCase):
+class TestIsInDPXTLCtxt (unittest.TestCase):
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:51]:
```suggestion
class TestIsInDPСTLCtxt (unittest.TestCase):
```

File: dpctl/tests/test_sycl_queue_manager.py [hunk 2]
```diff
@@ -19,51 +19,51 @@
...
 ##===----------------------------------------------------------------------===##
 
 import dpctl
 import unittest
 
 class TestGetNumPlatforms (unittest.TestCase):
-    @unittest.skipIf(not dpctl.has_sycl_platforms(), "No SYCL platforms available")
-    def test_dppl_get_num_platforms (self):
+    @unittest.skipIf(not dpctl.has_sycl_platforms(),
+                        "No SYCL platforms available")
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:30]:
```suggestion
    @unittest.skipIf(not dpctl.has_sycl_platforms(),
                     "No SYCL platforms available")
```

File: dpctl/tests/test_sycl_queue_manager.py [hunk 3]
```diff
@@ -19,51 +19,51 @@
...
 
     @unittest.skipIf(not dpctl.has_cpu_queues(), "No CPU platforms available")
-    def test_is_in_dppl_ctxt_inside_nested_device_ctxt (self):
+    def test_is_in_dpctl_ctxt_inside_nested_device_ctxt (self):
         with dpctl.device_context(dpctl.device_type.cpu):
             with dpctl.device_context(dpctl.device_type.gpu):
-                self.assertTrue(dpctl.is_in_dppl_ctxt())
-            self.assertTrue(dpctl.is_in_dppl_ctxt())
-        self.assertFalse(dpctl.is_in_dppl_ctxt())
+                self.assertTrue(dpctl.is_in_dpctl_ctxt())
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:64]: Why not to name function `is_in_device_context()` or `has_device_context()`?
Name `dpctl` is in module name. So it is known that the device context from dpctl in `dpctl.is_in_...()`.

File: dpctl/tests/test_sycl_queue_manager.py (hunk 1 above)
Comment (diptorupd) [reply dpctl/tests/test_sycl_queue_manager.py]: Fixed.

File: dpctl/tests/test_sycl_queue_manager.py (hunk 2 above)
Comment (diptorupd) [reply dpctl/tests/test_sycl_queue_manager.py]: Fixed.

File: dpctl/tests/test_sycl_queue_manager.py (hunk 3 above)
Comment (diptorupd) [reply dpctl/tests/test_sycl_queue_manager.py]: renamed to `is_in_device_context`

File: dpctl/tests/test_sycl_queue_manager.py [hunk 4]
```diff
@@ -19,51 +19,51 @@
...
 
-    def test_is_in_dppl_ctxt_inside_device_ctxt (self):
+    def test_is_in_dpctl_ctxt_inside_device_ctxt (self):
         with dpctl.device_context(dpctl.device_type.gpu):
-            self.assertTrue(dpctl.is_in_dppl_ctxt())
+            self.assertTrue(dpctl.is_in_device_context())
 
     @unittest.skipIf(not dpctl.has_cpu_queues(), "No CPU platforms available")
-    def test_is_in_dppl_ctxt_inside_nested_device_ctxt (self):
+    def test_is_in_dpctl_ctxt_inside_nested_device_ctxt (self):
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:61]:
```suggestion
    def test_is_in_device_context_inside_nested_device_ctxt (self):
```

File: dpctl/tests/test_sycl_queue_manager.py [hunk 5]
```diff
@@ -19,51 +19,51 @@
...
         except Exception:
             self.fail("Encountered an exception inside dump_device_info().")
 
 @unittest.skipIf(not dpctl.has_sycl_platforms(), "No SYCL platforms available")
-class TestDPPLIsInDPPLCtxt (unittest.TestCase):
+class TestIsInDeviceContext (unittest.TestCase):
 
-    def test_is_in_dppl_ctxt_outside_device_ctxt (self):
-        self.assertFalse(dpctl.is_in_dppl_ctxt())
+    def test_is_in_dpctl_ctxt_outside_device_ctxt (self):
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:53]:
```suggestion
    def test_is_in_device_context_outside_device_ctxt (self):
```

File: dpctl/tests/test_sycl_queue_manager.py [hunk 6]
```diff
@@ -19,51 +19,51 @@
...
+        self.assertFalse(dpctl.is_in_device_context())
 
-    def test_is_in_dppl_ctxt_inside_device_ctxt (self):
+    def test_is_in_dpctl_ctxt_inside_device_ctxt (self):
```
Comment (PokhodenkoSA) [inline dpctl/tests/test_sycl_queue_manager.py:56]:
```suggestion
    def test_is_in_device_context_inside_device_ctxt (self):
```

**Review by PokhodenkoSA — CHANGES_REQUESTED** [review]
Last fixes and we could merge it.

File: dpctl/tests/test_sycl_queue_manager.py (hunk 5 above)
Comment (diptorupd) [reply dpctl/tests/test_sycl_queue_manager.py]: Done. Thanks for catching it.

File: dpctl/tests/test_sycl_queue_manager.py (hunk 4 above)
Comment (diptorupd) [reply dpctl/tests/test_sycl_queue_manager.py]: Done. Thanks for catching it.

**Review by PokhodenkoSA — APPROVED** [review]

**Pull request merged**